packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -52,17 +62,21 @@ export default class OperatorModeOverlays extends Component<Signature> {
           data-test-overlay-selected={{if isSelected card.id}}
           data-test-overlay-card-display-name={{cardTypeDisplayName card}}
         >
-          {{! Add mouseenter and mouseleave events to each button, so we can maintain the hover effect. }}
           {{#if (this.isEmbeddedCard renderedCard)}}
             <div class='overlay-embedded-card-header' data-test-overlay-header>
-
-              {{! TODO: Icon for linksTo field type }}
               <div class='header-title'>
-                {{#if (eq renderedCard.fieldType 'contains')}}
-                  <div class='header-icon'>
+                <div class='header-icon'>
+                  {{#if (eq renderedCard.fieldType 'contains')}}
                     {{svgJar 'icon-turn-down-right' width='22px' height='18px'}}
-                  </div>
-                {{/if}}
+                  {{else}}
+                    {{#let (load (this.getRealmInfo card)) as |result|}}
```

Comment:
I found this neat `load` helper in the `ember-async-data` lib I added in this PR. It fetches async data straight from the template and it holds its own loading state.